requirements.txt

```diff
@@ -1,5 +1,5 @@
-PyInstaller==3.5
-typing==3.7.4.1
+PyInstaller>=4.6
+Pillow==9.2.0
```

Comment:
I like the change to `PyInstaller>=4.6`, but I'm confused by the change from `typing` (the type-checking tool) to `Pillow` (an image processing library). Was that an intentional change?

Reply:
Pillow is the python library to use the PIL image library package. I ended up with it for something, maybe PyInstaller. I will look back at it. `typing` is not required for python 3.9+ as it is included. As the current python is 3.10, I am trying to update this to work with that.

Reply:
I think I will get to the rest of these this weekend.

Reply:
> `typing` is not required for python 3.9+ as it is included

Oh I didn't know that. 👍 to removing the dependency then, thanks!

Reply:
Still have to figure out why I had to add Pillow. I think it is for icon building for MacOS.

- [x] Revert the scrolling frame changes

- [x] Revert the change from using / to join paths (if you do find this causes any issues, please open an issue and I can take another look at it)
- [x] Remove the test_ files -- I definitely agree with having more tests but we can open issues instead of adding TODO comments, that way all the work is tracked in one place
- [ ] Revert the requirements change from typing==3.7.4.1 to Pillow==9.2.0; this looks like a mistake (please correct me if I'm wrong!)